Tests/LoggingTests/MDCTest.swift

```diff
@@ -33,7 +33,7 @@ class MDCTest: XCTestCase {
             }
         }
         group.wait()
-        XCTAssertEqual(MDC.global["foo"], "bar", "expecting to find top items")
+        XCTAssertEqual(MDC.global["foo"]!, "bar", "expecting to find top items")
```

Comment:
why did this change?

Reply:
good spot, shouldn't be necessary anymore

Reply:
done